joomla/joomla-cms
# Use IpHelper to get IP in Logger

Pull Request for Issue #25518 .

### Summary of Changes

Use IpHelper to get IP in `Joomla\CMS\Log\Logger\FormattedtextLogger`.

### Testing Instructions

Install Joomla behind a reverse proxy which provides both remote_addr and http_x_forwarded_for headers.
Perform a login failure on /administrator
Check the /logs/error.php log

### Expected result

To see the ip address from http_x_forwarded_for reported as the offending IP address.

### Actual result

To see the ip address from remote_addr reported as the offending IP address.

### Documentation Changes Required

No.
